tests/integration/test_vulnerability_detector/test_scan_results/test_debian_inventory_debian_feeds.py

```diff
@@ -50,8 +50,13 @@ def get_configuration(request):
     """Get configurations from the module."""
     return request.param
 
+debian_systems = []
 
-@pytest.fixture(scope='module', params=debian_vulnerabilities)
+for debian_system in debian_vulnerabilities:
+    debian_systems.append(debian_system['target'])
+
```

Comment:
`debian_systems=[debian_systems['target'] for debian_system in debian_vulnerabilities]`

Reply:
Much better, thanks!

Reply:
https://github.com/wazuh/wazuh-qa/pull/801/commits/7cc23aa3aaf4ea3521a190a1ff754ffddef5c61a